### PR TITLE
Add shogun

### PR DESCRIPTION
**Pull request by bcai2**
Adds:
- `shogun_align` tool
- `shogun_filter` tool
- An integration test testing the `shogun_filter` -> `shogun_align` chain

### CODE REVIEW

File: toolchest_client/tools/api.py [hunk 1]
```diff
@@ -135,6 +135,72 @@ def kraken2(output_path, inputs=[], database_name="standard", database_version="
...
+
+    """
+
+    instance = ShogunFilter(
+        tool_args=tool_args,
+    output_name='output.txt',  # TODO: add actual outputs
+        inputs=inputs,
+        output_path=output_path,
+        database_name=database_name,
+        database_version=database_version
```
Comment (lebovic) [inline toolchest_client/tools/api.py:199]: LGTM, is this added in the staging db yet?

File: toolchest_client/tools/api.py [hunk 2]
```diff
@@ -135,6 +135,72 @@ def kraken2(output_path, inputs=[], database_name="standard", database_version="
...
+    )
+    instance.run()
+
+
+def shogun_filter(inputs, output_path, database_name="shogun_standard", database_version="1", tool_args=""):
+    """Runs Shogun (for filtering human genome content) via Toolchest.
+
+    :param tool_args: (optional) Additional arguments to be passed to Shogun.
+    :param database_name: (optional) Name of database to use for Shogun alignment. Defaults to the pre-built DB files at https://github.com/knights-lab/SHOGUN.
+    :param database_version: (optional) Version of database to use for Bowtie 2 alignment.
```
Comment (lebovic) [inline toolchest_client/tools/api.py:176]: Typo ("Bowtie 2")

File: toolchest_client/tools/api.py [hunk 3]
```diff
@@ -135,6 +135,72 @@ def kraken2(output_path, inputs=[], database_name="standard", database_version="
     instance.run()
 
 
+def shogun_align(inputs, output_path, database_name="shogun_standard", database_version="1", tool_args=""):
+    """Runs Shogun (for alignment) via Toolchest.
+
+    :param tool_args: (optional) Additional arguments to be passed to Shogun.
+    :param database_name: (optional) Name of database to use for Shogun alignment. Defaults to the pre-built DB files at https://github.com/knights-lab/SHOGUN.
+    :param database_version: (optional) Version of database to use for Bowtie 2 alignment.
```
Comment (lebovic) [inline toolchest_client/tools/api.py:143]: Typo ("Bowtie 2")

**Review by lebovic — APPROVED** [review]

File: toolchest_client/tools/api.py (hunk 1 above)
Comment (bcai2) [reply toolchest_client/tools/api.py:199]: Yes :+1:

File: toolchest_client/tools/api.py (hunk 2 above)
Comment (bcai2) [reply toolchest_client/tools/api.py:176]: Oops, ty

File: toolchest_client/tools/api.py (hunk 3 above)
Comment (bcai2) [reply toolchest_client/tools/api.py:143]: Oops again, ty

**Pull request merged**